sunpy-core.md

```diff
@@ -6,4 +6,8 @@ Planned release by June 2019 with associated paper. The details of this release
 
 ## 2.0
 
+* Create a generic command to read in files such as `sunpy.open()` or `sunpy.read()`
```

Comment:
:-1: 

I am really not in favour of this, there is way way way too much unpredictability in the data sources we support to make this not have 1001 weird and wonderful edge cases.

Reply:
how else would we be able to support arbitrary data types with ndcube without a generic reader?

Reply:
I am not sure NDCube as it stands can support "arbitary data types", NDCube requires you have a WCS or gWCS object to work. Only "Map" like FITS ever really have those at the moment. I have never seen a timeseries with a WCS.

Reply:
Spectral data like IRIS uses WCS. I agree that time series may not make sense. This obviously needs more discussion.